lib/property.js

```diff
@@ -122,9 +122,12 @@ class Property {
    */
   getValue() {
     return new Promise((resolve) => {
-      console.log('getValue for property', this.name,
-                  'for:', this.device.name,
-                  'returning', this.value);
+      if (this.value != this.prevGetValue) {
+        console.log('getValue for property', this.name,
+                    'for:', this.device.name,
+                    'returning', this.value);
+        this.prevGetValue == this.value;
```

Comment:
s/==/=/

Reply:
Bah - that's what I get for not actually testing this :)